boilerplate/openshift/golang-osd-operator/csv-generate/common-generate-operator-bundle.py

```diff
@@ -65,6 +86,8 @@
                 print(
                     f"Loading {doc['kind']} {doc['metadata']['name']} " +
                     f"from {path}")
+                if doc['kind'] not in BUNDLE_PERMITTED_RESOURCES:
+                    raise UnsupportedRegistryResourceKind(doc['kind'], path)
```

Comment:
I'm testing this locally rn, but I think this will blow up because `UnsupportedRegistryResourceKind` isn't defined yet.

/hold

Reply:
Okay, yeah, couple problems here.
1. As noted, this [raises `NameError` rather than `UnsupportedRegistryResourceKind`](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_boilerplate/149/pull-ci-openshift-boilerplate-master-pr-check/1383065847995568128#1:build-log.txt%3A4008) because the latter isn't defined yet.
2. It's [raising it on the `Deployment` resource](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_boilerplate/149/pull-ci-openshift-boilerplate-master-pr-check/1383065847995568128#1:build-log.txt%3A4004). We should add that to the supported list with a comment that it's processed specially by the script and not included in the bundle.